src/site/stages/build/process-cms-exports/schemas/common/generic-nested-string.js

```diff
@@ -6,5 +6,7 @@ module.exports = {
     properties: {
       value: { type: 'string' },
     },
+    required: ['value'],
   },
+  maxItems: 1,
```

Comment:
The pattern we've seen is an array of one item with an on object containing a `value` key.  I'd like to be notified if we ever have more than one item in this array.

Reply:
Good call on both these :+1: